src/scripts/events/domEvents.js

```diff
@@ -5,41 +5,39 @@ import { deleteAuthorBooks, viewBookDetails } from '../../api/mergedData';
 import { showAuthors } from '../components/pages/authors';
 import addBookForm from '../components/forms/addBookForm';
 import addAuthorForm from '../components/forms/addAuthorForm';
-import { updateAuthor } from '../../api/authorData';
+import { getSingleAuthor } from '../../api/authorData';
 // import selectAuthor from '../components/forms/selectAuthor';
 import viewBook from '../components/pages/viewBook';
 // import { showAuthors } from '../components/pages/authors';
 
-const domEvents = () => {
+const domEvents = (uid) => {
   document.querySelector('#main-container').addEventListener('click', (e) => {
     // TODO: CLICK EVENT FOR DELETING A BOOK
     if (e.target.id.includes('delete-book')) {
       // eslint-disable-next-line no-alert
       if (window.confirm('Want to delete?')) {
         const [, firebaseKey] = e.target.id.split('--');
 
-        deleteBook(firebaseKey).then((booksArray) => showBooks(booksArray));
+        deleteBook(firebaseKey, uid).then((booksArray) => showBooks(booksArray));
```

Comment:
Please sure to include the uid parameter in your deleteBook API call

Reply:
Thanks, @ajawashington ! 🙏